### ISSUE TITLE
CSS of the mifes are missing on the deployment

### ISSUE THREAD

**Issue by mark-avd** [WJSoftware/vite-plugin-single-spa#21]
Hello, Jose.
First and foremost, I would like to thank you for creating this plugin. You saved me from migrating to webpack :D 
And now, onto the issue:

Root app is deployed on `example.com` and mife on `example.com:4101`.
importMap.json of the mife:
```
{
  "imports": {
    "@mife/navigation": "http://example.com:4101/spa.js",
  }
}
```
vite.config.ts:
```
plugins: [
  react(),
  viteTsconfigPaths(),
  vitePluginSingleSpa({
    type: 'mife',
    serverPort: 4101,
    projectId: 'navigation',
  }),
],
```
The problem is that the CSS of the mife is being requested from the deployment of the root application instead of the deployment of the mife itself.
In my case in browser devtools I can see that app trying to get mife's CSS from the `http://example.com/assets/vpss(navigation)spa-240be869.css` instead of `http://example.com:4101/assets/vpss(navigation)spa-240be869.css` where it actually is.

Also, in version 0.1.0 of the plugin, there was an option called `deployedBase`, which allowed to solve this issue by specifying in my case:
```
vitePluginSingleSpa({
  type: 'mife',
  serverPort: 4101,
  deployedBase: 'http://example.com:4101',
 }),
```

**Comment by webJose** [comment]
Hello!

> You saved me from migrating to webpack

I was in a much worse position:  Create React App (with webpack) and wanted out badly!

You are on the right track:  The base.  Remember that Vite provides the `base` property.  Not me.  Just set it.  I recommend doing it like this:

```javascript
export default function ({ command }) {
    return defineConfig({
        plugins: [
            react(),
            viteTsconfigPaths(),
            vitePluginSingleSpa({
                type: 'mife',
                serverPort: 4101,
                projectId: 'navigation',
            }),
        ],
        base: command === 'build' ? 'http://example.com:4101' : undefined
    });
};
```

I used to have `deployedBase` because I was using it for two things.  When I discovered `server.origin`, then I figured my need was reduced drastically, so I dropped it.

In retrospective, seeing that I am forced to make the export as a function and only set the base when building, I might reconsider reviving `deployedBase`, as my plug-in would only apply it as the name says:  When deploying (a. k. a., building).

On the other hand, I just know that I cannot free every developer out there from having to use the function version of Vite config, so my brain tells me that I shouldn't overextend. 

I'll leave the issue open for now.  Feel free to close it if this solves your problem.
